GiniVisionExample/GiniVisionExample/ViewControllers/HelpViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  HelpViewController.swift
+//  GiniVisionExample
+//
+//  Created by Enrique del Pozo Gómez on 11/21/17.
+//  Copyright © 2017 Gini. All rights reserved.
+//
+
+import UIKit
+import GiniVision
+
+typealias HelpLink = (title: String, url: URL?)
+typealias HelpVersion = (title: String, version: String)
+
+protocol HelpViewControllerDelegate: class {
+    func help(viewController: HelpViewController, didSelectLink link: HelpLink)
+    func help(viewController: HelpViewController, didTapClose: ())
+}
+
+final class HelpViewController: UIViewController {
+    weak var delegate: HelpViewControllerDelegate?
+    let linkCellReuseIdentifier = "linkCellReuseIdentifier"
+    let versionCellReuseIdentifier = "versionCellReuseIdentifier"
+    
+    let versions: [HelpVersion] = [("GVL Version", AppHelper.gvlVersion ?? ""),
+                                   ("App Version", AppHelper.appVersion ?? "")]
+    let links: [HelpLink] = [("GVL Changelog",
+                              URL(string: "http://developer.gini.net/gini-vision-lib-ios/docs/changelog.html")),
```

Comment:
Yes, I see now why you needed to disable the transport security. But even if you HAVE to use that exact URL, you should probably just add that domain as an exception.

Additionally, using Apple's browser window view controller, might solve the problem. See below.